app/services/force/listing_service.rb

```diff
@@ -27,9 +27,20 @@ def subqueries
       %(
       	(SELECT Date__c, Start_Time__c, End_Time__c, Venue__c, Street_Address__c, City__c FROM Information_Sessions__r),
       	(SELECT Date__c, Start_Time__c, End_Time__c FROM Open_Houses__r),
-      	(SELECT Id, Total_Submitted_Apps__c, Order__c, Description__c, Available_Units__c, PDF_URL__c, Lottery_Preference__r.Id, Lottery_Preference__r.Name FROM Listing_Lottery_Preferences__r),
-      	(SELECT Unit_Type__c, BMR_Rent_Monthly__c, BMR_Rental_Minimum_Monthly_Income_Needed__c, Status__c, Property_Type__c, AMI_chart_type__c, AMI_chart_year__c, of_AMI_for_Pricing_Unit__c, Reserved_Type__c FROM Units__r ORDER BY Unit_Type__c)
+      	(SELECT Id, Total_Submitted_Apps__c, Order__c, Description__c, Available_Units__c, PDF_URL__c, Lottery_Preference__r.Id,
+                Lottery_Preference__r.Name FROM Listing_Lottery_Preferences__r),
+      	(SELECT Unit_Type__c, BMR_Rent_Monthly__c, BMR_Rental_Minimum_Monthly_Income_Needed__c, Status__c, Property_Type__c,
+                AMI_chart_type__c, AMI_chart_year__c, of_AMI_for_Pricing_Unit__c, Reserved_Type__c FROM Units__r ORDER BY Unit_Type__c)
       )
     end
+
+    def units(listing_id)
+      parsed_index_query(%(
+        SELECT Unit_Type__c, BMR_Rent_Monthly__c, BMR_Rental_Minimum_Monthly_Income_Needed__c, Status__c, Property_Type__c,
```

Comment:
According to Rachel we should be using Listing Unit when our goal is to get all units that belong to a listing. The Listing Unit object is currently used to represent the relationship between a Listing and a Unit. However, it is then unclear why Units have a field for the listing ID - that makes it seem like the relationship object isn't actually needed. This raises the question of whether the true relationships between listings and units is represented by a) the Listing Unit objects, or b) the listing id field on the Unit object. We haven't dealt directly with this that much yet because the web app uses a custom API call to fetch listing units. I'm following up further with Rachel on this to understand the exact distinctions and to see if there is duplicate functionality present that could be removed to clarify. Until then, my inclination would be to use Rachel's recommendation to use Listing Unit until we have more info suggesting otherwise, and just query and pass the fields needed to fulfill the AC in this story.

Reply:
Have not heard any more info from Rachel yet, but I looked further into where Unit and Listing Unit are used, and I found that Leases appear to only have a relationship with Units, not with Listing Units. This doesn't say anything about whether the listing-unit relationship is managed through Listing Units or through a `Unit__c` field. But it does provide another example of where it seems to be more convenient to use Units directly. Based on this, I now feel fine with sticking with using Units directly in this PR. I will still follow up with Rachel so we can get clarity on this issue.

Reply:
looks like mapUnits expects all of those fields to be there :(

Reply:
ok then let's go ahead with including those fields